test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] When a file is ignored by GitHub (`added in the .gitignore file`), it should not be present on the repository either, you have done fine as said by the previous reviewers but the issue is your test.md file is still visible on the repository, consider running this command `git rm test.md to remove it from the repo and you will be good to go afterward.